install_config/aggregate_logging.adoc

```diff
@@ -1868,15 +1807,15 @@ systemctl restart docker
 
 . Update the xref:fluentd-upgrade-source[Fluentd log source].
 
+. Restart the Fluentd:
++
+----
+systemctl restart fluentd
```

Comment:
What is the correct command to restart Fluentd? I copied the command to restart Docker as a placeholder. 

I found the following in Fluentd docs but don't think it's correct:
"sudo service td-agent restart"

Reply:
NOTE: Restarting fluentd on many nodes at once will create a huge load on the Kubernetes scheduler, so exercise caution about following the directions below if you have more than a few dozen nodes.

To restart fluentd on *all* nodes, use `oc label node -l logging-infra-fluentd=true --overwrite logging-infra-fluentd=false` to shutdown fluentd on all nodes where it is running, then use `oc get ds -l component=fluentd` to make sure they have all been shutdown, then `oc label node -l logging-infra-fluentd=false --overwrite logging-infra-fluentd=true` to restart fluentd on those nodes.

To restart fluentd on one or a set of nodes:
`oc get nodes -l logging-infra-fluentd=true` to list the nodes where fluentd is running.  Then for each node, use `oc label node $node logging-infra-fluentd-` to unlabel the node and turn off fluentd.  Use `oc get ds -l component=fluentd` to see if those fluentd have been shutdown.  Then for each node, use `oc label node $node logging-infra-fluentd=true` to start up fluentd on those nodes.

Reply:
Thanks, @richm! I updated the steps and added a warning about the load on the Kubernetes scheduler.